src/main/groovy/org/transmartproject/core/dataquery/clinical/ClinicalVariable.groovy

```diff
@@ -84,4 +82,12 @@ public interface ClinicalVariable {
      * or {@link BigDecimal}.
      */
     public final static String NORMALIZED_LEAFS_VARIABLE = 'normalized_leafs_variable'
+
+    /**
+     * The hierarchical path that leads to the concept.
+     *
+     * @return the full path of the concept; path with \ as a separator
+     */
+    public String getConceptPath()
```

Comment:
So you go through the trouble of introducing `ConceptKey` and `ConceptFullName` to the API and then end up using strings to represent the paths?